configure.ac

```diff
@@ -166,12 +166,26 @@ AC_CHECK_TOOLS(QMAKE, [qmake-qt5 qmake], "no")
 AC_CHECK_TOOLS(LUPDATE, [lupdate-qt5 lupdate])
 AC_CHECK_TOOLS(LRELEASE, [lrelease-qt5 lrelease])
 
+MIN_QT_VERSION=5.7
```

Comment:
Please make a note here and in main.cc and gui/main.cc to tie the QT_VERSION_CHECK there and MIN_QT_VERSION here together to keep them in sync.

[ Talking aloud. ] Why are both necessary? This lets the UNIXy "everything is ./configure && make' users fail fast and the other is needed for the Qt "load project, press build" users.

Reply:
A version check could go into GPSBabel.pro and gui/app.pro, but it would be nice to use versionAtLeast() and that may not exist in the version the developer is trying to use.

Also, we have/had other methods besides configure and .pro, e.g. xcode, and perhaps some old msvc files somewhere?